index.coffee

```diff
@@ -8,15 +8,20 @@ exec = require('child_process').exec
 gutil = require 'gulp-util'
 through = require 'through2'
 
-module.exports = (options = {}) ->
+module.exports = (options = {bundle:true}) ->
 
```

Comment:
I found an issue with the "default option" object. If you set an option object without the bundle property included, bundle will be false by default. "bundle" should always be true except someone set "bundle: false" explicitly.